docs/development.rst

```diff
@@ -63,11 +63,11 @@ To watch files and build bundles automatically, use this.
 Configure
 .........
 
-Configuration is done via environment variables. For the meaning of them, look them up under files in ``config/settings``. Values in the file ``.env`` will be used automatically.
+Configuration is done via environment variables. For the meaning of them, look them up under files in ``config/settings``. Values in ``env.local`` will be used automatically.
```

Comment:
This is reverting my previous commit. I think you somehow managed to copy the old file with your changes, thus reverting some other stuff I committed since. This is why it is important to develop in branches, not in the master branch.